common/errors_test.go

```diff
@@ -113,6 +115,7 @@ This is a very very long response body.`
 	r.WriteHeader(http.StatusUnauthorized)
 
 	resp := r.Result()
+	defer resp.Body.Close()
```

Comment:
This change has been fixed as part of separate pr - https://github.com/anz-bank/sysl-go/pull/253